Meta/Lagom/CMakeLists.txt

```diff
@@ -25,6 +25,8 @@ get_filename_component(
 )
 set(SerenityOS_SOURCE_DIR "${SERENITY_PROJECT_ROOT}" CACHE STRING "")
 
+set(SERENITY_CACHE_DIR "${PROJECT_BINARY_DIR}" CACHE STRING "")
```

Comment:
These look like the correct changes. I think it's missing one thing still, which is the interaction with `Meta/serenity.sh test lagom` when there's not already a Lagom build dir created by a super build. 

Not sure how many people are running into that edge case though.

Reply:
Bah, I had a serenity.sh change to cover that case, would've helped if I commited it :expressionless: